### PR TITLE
Adding the second key from the topic 0

### PR DESCRIPTION
**Pull request by lucianoext**
I still have to implement the gitignore.

### CODE REVIEW

File: bootcamp/src/topic0/DuplexBuilder.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+		@Override
+		public void buildAddress() {
+			building.setAddressNumber(42);;
+			building.setStreet("St Monica");
+		}
+		@Override
+		
+		public void buildStructure(){
+			building.setNFloors(2);
```
Comment (tomiok) [inline bootcamp/src/topic0/DuplexBuilder.java]: At this code, we are asuming that ALWAYS adding that number or floors, rooms and the same address for all the buildings... need to pass this as a param, in order to be more flexible

**Comment by lucianoext** [comment]
sure, so should I pass the atributes from the demo ?

**Comment by lucianoext** [comment]
Now that I see it with the changes there is no point in having home and
duplex as differents products since the only thing that differentiated them
were their attributes.Should I add another attribute that specify the type
of building that it is?

**Comment by lucianoext** [comment]
I pushed a new version for the second key from the topic 0, I'm having
 deleting the previous version from the remote repo.

File: t0k2/src/building/Address.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+public class Address {
+	private String street;
+	private int addressNumber;
+	
+	@Override
+	public String toString() {
+		return "Address [street=" + street + ", addressNumber=" + addressNumber + "]";
+	}
+
+	private Address(Builder b){
```
Comment (juanipaccapelo-globant) [inline t0k2/src/building/Address.java]: Here you are coupling the Address to the builder. The Address shouldn't know anything about the builder but the other way around

**Comment by lucianoext** [comment]
I've just pushed the topic 0 key 3, in which I am having some troubles.
I'm working on separating the Address from its builder.

**Comment by lucianoext** [comment]
I've just pushed the key 2 and 3 from the topic 0.

**Comment by lucianoext** [comment]
closing

**Pull request closed without merging**